src/cli/grain.js

```diff
@@ -59,28 +62,34 @@ const grainCommand: Command = async (args, std) => {
   }
 
   console.log(
+    simulation ? `——SIMULATED DISTRIBUTION——\n` : ``,
     `Distributed ${G.format(totalDistributed)} to ${
       recipientIdentities.size
     } identities in ${distributions.length} distributions`
   );
 
-  await fs.writeFile(ledgerPath, ledger.serialize());
+  if (!simulation) {
+    await fs.writeFile(ledgerPath, ledger.serialize());
 
-  const credAccounts = computeCredAccounts(ledger, credView);
-  const accountsPath = join(baseDir, "output", "accounts.json");
-  await fs.writeFile(accountsPath, stringify(credAccounts));
+    const credAccounts = computeCredAccounts(ledger, credView);
+    const accountsPath = join(baseDir, "output", "accounts.json");
+    await fs.writeFile(accountsPath, stringify(credAccounts));
+  }
 
   return 0;
 };
 
 export const grainHelp: Command = async (args, std) => {
   std.out(
     dedent`\
-      usage: sourcecred grain
+      usage: sourcecred grain [--simulation || -s]
 
       Distribute Grain (or whatever currency this Cred instance is tracking)
       for Cred intervals in which Grain was not already distributed.
 
+      When the '--simulation' (-s) flag is provided, the instance's ledger and accounts
+      data will not be written to, allowing for testing the output of various configurations.
```

Comment:
suggest saying "no Grain will actually be distributed" rather than "the instance's ledger and accounts data will not be written to". It's clearer to explain the behavior rather than explaining the implementation. :)